payments-contracts/src/main/kotlin/com/gevamu/corda/contracts/PaymentContract.kt

```diff
@@ -20,6 +20,9 @@ import net.corda.core.contracts.CommandData
 import net.corda.core.contracts.Contract
 import net.corda.core.transactions.LedgerTransaction
 
+/**
```

Comment:
It's good to describe verification logic. Please speak with Andrey Senov and add it to the Pull Request which adds verification and on which Andrey's working on